README.md

```diff
@@ -89,6 +89,10 @@ const style = css({
 })
 ```
 
+### Images and other static files
+
+Create a folder called `static` in your project root directory. You can place images and other static assets there. Reference them from JS files with the path to the static directory. For example, from `./components` use `../static/file-name.jpg` or from `./pages` simpy use `/static/file-name.jpg`
```

Comment:
I think we should not add the example of`./components` since it's not the directory next.js requires,
and the best practice of wring file paths in HTML would be to use absolute paths (no `..`) IMHO.